geopandas/tests/test_array.py

```diff
@@ -922,7 +909,8 @@ def test_estimate_utm_crs__geographic(self):
                 self.landmarks.estimate_utm_crs()
         else:
             assert self.landmarks.estimate_utm_crs() == CRS("EPSG:32618")
-            assert self.landmarks.estimate_utm_crs("NAD83") == CRS("EPSG:26918")
+            if compat.PYPROJ_GE_32:  # result is unstable in older pyproj
+                assert self.landmarks.estimate_utm_crs("NAD83") == CRS("EPSG:26918")
```

Comment:
Didn't we test with such older version before, or is it triggered due to the combo with a certain fiona install?

Reply:
No. We tested with 2.2 which skipped the whole test and latest. I have added 3.1 which causes the issue #2045